permissions/resource_permissions.go

```diff
@@ -176,6 +174,21 @@ func (a PermissionsAPI) Update(objectID string, objectACL AccessControlChangeLis
 			PermissionLevel: "CAN_MANAGE",
 		})
 	}
+	w, err := a.client.WorkspaceClient()
+	if err != nil {
+		return err
+	}
+	if strings.HasPrefix(objectID, "/sql/warehouses") {
+		warehouse, err := w.Warehouses.GetById(a.context, strings.ReplaceAll(objectID, "/sql/warehouses/", ""))
```

Comment:
for debuggability, could you extract `strings.ReplaceAll(objectID, "/sql/warehouses/", "")` to a variable?